easybuild/easyconfigs/s/squashfs-tools/squashfs-tools-4.6.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
+
+name = 'squashfs-tools'
+version = '4.6.1'
+
+homepage = ' https://github.com/plougher/squashfs-tools'
+description = """squashfs-fuse"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/plougher/squashfs-tools/archive']
+sources = ['master.tar.gz']
+
+build_cmd = "cd squashfs-tools ; make"
+
+
+files_to_copy = [
+    (['squashfs-tools/*'], 'bin'),
+    (['squashfs-tools/*.h'], 'include'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/mksquashfs', 'bin/unsquashfs'],
+    'dirs': [],
+}
+
+modextrapaths = {'PATH': ' '}
```

Comment:
this should be removed as the binaries are in `bin`, which is automatically added to the PATH by EB:
```suggestion
```